ext/snmp/tests/snmp-object-properties-error.phpt

```diff
@@ -28,6 +28,12 @@ try {
     echo $exception->getMessage() . "\n";
 }
 
+try {
+    $session->max_oids += 1;
```

Comment:
I haven't manage to install SNMP locally, so I have to wait for the CI results. :S